### PR TITLE
Cosmetic corrections

### PR DESCRIPTION
**Pull request by gshimansky**

### CODE REVIEW

**Comment by ifilippov** [comment]
dst-mac, really? (approved)

**Comment by ifilippov** [comment]
To develop branch

File: examples/nat/main/config.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
             },
             "public-port": {
                 "index": 1,
-                "dst_mac": "3c:fd:fe:a4:d3:b1",
+                "dst-mac": "3c:fd:fe:a4:d3:b1",
```
Comment (mbulatova) [inline examples/nat/main/config.json]: Maybe remaining dst_mac -> dst-mac too?

Comment (gshimansky) [reply examples/nat/main/config.json]: dst_mac on private port should be deleted entirely since it is not used at all any more. Thanks for catching this.

**Comment by gshimansky** [comment]
This change should not go into master yet

**Pull request closed without merging**